CancelBooking UseCase/ReservationTest.java

```diff
@@ -1,6 +1,7 @@
 import org.junit.Test;
 import java.time.LocalDate;
 import static org.junit.Assert.*;
+

```

Comment:
```suggestion
the (Reservation) class has getters like (getCustomerName()), but in the test, you are calling (getcustomerName()) and it would be better to ensure consistent naming. Also, you can add test edge cases such as the scenario when invalid data is passed to the constructor.
```